media-proxy/src/mesh/logger.cc

```diff
@@ -113,21 +115,24 @@ void setFormatter(std::unique_ptr<Formatter> new_formatter)
     formatter = std::move(new_formatter);
 }
 
+void setLogLevel(Level level) {
+    currentLogLevel = level;
+}
```

Comment:
This function is not thread-safe. Since there is no meaning in changing the log level at runtime, I suggest to leave this function as it is but say in the comment block that this is to be called only once the logger is created in the app.

Reply:
Actually there was a meaning for me - for different tests I wanted to have control over what is visible where, I'll make it thread safe.

Reply:
If you make it thread safe, you'll make `currentLogLevel` thread safe. It will add latency to printing messages. Please don't.